compose.dev.yaml

```diff
@@ -37,6 +37,20 @@ services:
       - ./docker/flipdot.schema:/assets/openldap/custom/schema/flipdot.schema
       - ./docker/flipdot.ldif:/assets/openldap/custom/ldif/flipdot.ldif
 
+  keycloak:
+    image: "quay.io/keycloak/keycloak:25.0.4"
+    environment:
+        KEYCLOAK_ADMIN: admin
+        KEYCLOAK_ADMIN_PASSWORD: admin
```

Comment:
This should be read form a `.env` file, but I guess that's already on your radar :wink:

Reply:
Keycloak in compose is only needed for dev setup. I see no need to configure it via env vars

Reply:
Ah, I see. Is there another Keycloak configured for production already? Didn't know of any yet.

Reply:
Yes, checkout https://login.flipdot.org/, you can login with your existing flipdot LDAP credentials :) Some of our applications already support a login with it, checkout the "Applications" section in keycloak after login (door not working yet, the rest of the applications do, but some need manual linking or migration effort beforehand). This now allows you to use passkeys to login at various flipdot services!

Reply:
Wow, nice! :fire: I smell some incoming talks between us about this architecture in October :wink: